tuning/sft_trainer.py

```diff
@@ -651,6 +655,25 @@ def main():
             )
             sys.exit(INTERNAL_ERROR_EXIT_CODE)
 
+    # post process lora
+    if isinstance(tune_config, peft_config.LoraConfig):
+        try:
+            checkpoint_dir = job_config.get("save_model_dir")
+            if not checkpoint_dir:
+                checkpoint_dir = os.path.join(
+                    training_args.output_dir,
+                    get_highest_checkpoint(training_args.output_dir),
+                )
+            print(training_args)
```

Comment:
is print(training_args) added on purpose?

Reply:
No, thanks